android/sdl_android/src/androidTest/java/com/smartdevicelink/test/VehicleDataHelper.java

```diff
@@ -228,7 +228,7 @@
 
     static {
         //TIRE_PRESSURE set up
-        TIRE_PRESSURE.setPressureTellTale(TIRE_PRESSURE_TELL_TALE);
```

Comment:
@kostyaBoss I would suggest reverting these changes in `VehicleDataHelper.java`.  These are valid changes, since the `pressureTellTale` functions are being renamed per the parameter, but is out of scope of the PR.